vms/platformvm/txs/executor/import_test.go

```diff
@@ -36,63 +38,16 @@ func TestNewImportTx(t *testing.T) {
 	sourceKey, err := secp256k1.NewPrivateKey()
 	require.NoError(t, err)
 
-	cnt := new(byte)
-
-	// Returns a shared memory where GetDatabase returns a database
-	// where [recipientKey] has a balance of [amt]
-	fundedSharedMemory := func(peerChain ids.ID, assets map[ids.ID]uint64) atomic.SharedMemory {
```

Comment:
this a clean way to setup the shared memory when we want to build an importTx. Pulled the function out of the test to be reused in other tests